### PR TITLE
Add `--all-rows` Flag

### PR DESCRIPTION
**Pull request by zliang-akamai**
## 📝 Description

Provide a option for user to output all results without pagination.

Should we add some integration/unit tests for this? Will be working on that if necessary.

## ✔️ How to Test

Create 1000 stackscripts in your account then run `linode --all-rows stackscripts ls --is_public false`

### CODE REVIEW

File: linodecli/cli.py [hunk 1]
```diff
@@ -479,6 +482,59 @@ def _flatten_url_path(tag):
...
+
+            result = self.merge_results_data(
+                itertools.chain(
+                    (result,),
+                    self._get_all_pages_results_generator(
+                        operation, args, pages_needed
+                    ),
+                )
+            )
+        return result
```
Comment (lgarber-akamai) [inline linodecli/cli.py:536]: This looks great! Could we add some unit tests to make sure all of these function work as expected with some mock data?

Comment (zliang-akamai) [reply linodecli/cli.py]: Thank you! Test added.

File: linodecli/__init__.py [hunk 2]
```diff
@@ -77,8 +77,16 @@ def main():  # pylint: disable=too-many-branches,too-many-statements
         cli.output_handler.columns = "*"
     if parsed.no_headers:
         cli.output_handler.headers = False
-    if parsed.all:
+    if parsed.all_columns or parsed.all:
+        if parsed.all:
```
Comment (lgarber-akamai) [inline linodecli/__init__.py:81]: Could we check whether warnings should be suppressed here?

**Review by lgarber-akamai — CHANGES_REQUESTED** [review]
Works as expected on my end. One small change and we should be good to go 👍

**Review by lgarber-akamai — APPROVED** [review]

**Pull request merged**